composer.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "friendsofcrawler/typo3-crawler-widget",
+  "type": "typo3-cms-extension",
+  "description": "Widgets for crawler extension",
+  "homepage": "https://github.com/tomasnorre/typo3-crawler-widget",
+  "support": {
+    "issues": "https://github.com/tomasnorre/typo3-crawler-widget/issues"
+  },
+  "license": "GPL-3.0",
```

Comment:
This doesn't match with the licens information in the PHP Files or the LICENSE file. 

Composer say: GPL3
PHP: GPL2+
LICENSE: MIT

We need to clean that up :)

Reply:
Do you have any suggestion? I am not really familiar with all those licences.

Reply:
GPL2+ or MIT, GPL3 is not compliant with the TYPO3 Core.

Reply:
I decided to use GPL2+